test/e2e/apimachinery/aggregator.go

```diff
@@ -69,7 +69,10 @@ var _ = SIGDescribe("Aggregator", func() {
 	BeforeEach(func() {
 		c = f.ClientSet
 		ns = f.Namespace.Name
-		aggrclient = f.AggregatorClient
+		config, err := framework.LoadConfig()
```

Comment:
This is pretty weird to local load vs using one of the other clients that already exist in the framework.  Why do you need to load this separately?

Reply:
i don't see the config persisting in the framework, just the clients.
but here were removing a client from the framework and creating it in a test, thus we also need a config.

alternative ideas are welcome.

Reply:
i will move the initialization away from BeforeEach.